Errores a corregir en el travis

FLAKE8
```
crm_campaign_configuration/__manifest__.py:14:13: E126 continuation line over-indented for hanging indent
```

Pylint
```
crm_campaign_configuration/views/res_partner_views.xml:8: [W7910(wrong-tabs-instead-of-spaces), ]  Use wrong tabs indentation instead of four spaces
crm_campaign_configuration/views/res_partner_views.xml:9: [W7910(wrong-tabs-instead-of-spaces), ]  Use wrong tabs indentation instead of four spaces
crm_campaign_configuration/views/res_partner_views.xml:10: [W7910(wrong-tabs-instead-of-spaces), ]  Use wrong tabs indentation instead of four spaces
```

--- a/crm_campaign_configuration/README.rst
+++ b/crm_campaign_configuration/README.rst
@@ -1,0 +1,28 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==========================
+CRM campaign configuration
+==========================
+
+* Campaign medium and source in res partner.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/avanzosc/odoo-addons/issues>`_. In case of trouble,
+please check there if your issue has already been reported. If you spotted
+it first, help us smash it by providing detailed and welcomed feedback.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Berezi Amubieta <bereziamubieta@avanzosc.es>
+* Alfredo de la Fuente <alfredodelafuente@avanzosc.es>

--- a/crm_campaign_configuration/__init__.py
+++ b/crm_campaign_configuration/__init__.py
@@ -1,0 +1,1 @@
+from . import models

--- a/crm_campaign_configuration/__manifest__.py
+++ b/crm_campaign_configuration/__manifest__.py
@@ -1,0 +1,24 @@
+# Copyright 2021 Berezi - Iker - AvanzOSC
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "CRM campaign configuration",
+    'version': '14.0.1.0.0',
+    "author": "Avanzosc",
+    "category": "Sales/CRM",
+    "depends": [
+        "contacts",
+        "crm",
+        "utm"
+    ],
+    "data": [
+        "views/res_partner_views.xml",
+        "views/utm_source_views.xml",
+        "views/utm_campaign_views.xml",
+        "views/utm_medium_views.xml",
+        "views/contact_medium_views.xml",
+        "views/contact_source_views.xml",
+        "views/contact_campaign_views.xml"
+    ],
+    "license": "AGPL-3",
+    'installable': True,
+}

--- a/crm_campaign_configuration/i18n/crm_campaign_configuration.pot
+++ b/crm_campaign_configuration/i18n/crm_campaign_configuration.pot
@@ -1,0 +1,60 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* crm_campaign_configuration
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-05-28 08:49+0000\n"
+"PO-Revision-Date: 2021-05-28 08:49+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: crm_campaign_configuration
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_partner__campaign_id
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_users__campaign_id
+#: model:ir.ui.menu,name:crm_campaign_configuration.menu_contact_campaign
+#: model:ir.ui.menu,name:crm_campaign_configuration.menu_crm_utm_campaign
+msgid "Campaign"
+msgstr ""
+
+#. module: crm_campaign_configuration
+#: model:ir.model,name:crm_campaign_configuration.model_res_partner
+msgid "Contact"
+msgstr ""
+
+#. module: crm_campaign_configuration
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_partner__display_name
+msgid "Display Name"
+msgstr ""
+
+#. module: crm_campaign_configuration
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_partner__id
+msgid "ID"
+msgstr ""
+
+#. module: crm_campaign_configuration
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_partner____last_update
+msgid "Last Modified on"
+msgstr ""
+
+#. module: crm_campaign_configuration
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_partner__medium_id
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_users__medium_id
+#: model:ir.ui.menu,name:crm_campaign_configuration.menu_contact_medium
+#: model:ir.ui.menu,name:crm_campaign_configuration.menu_crm_utm_medium
+msgid "Medium"
+msgstr ""
+
+#. module: crm_campaign_configuration
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_partner__source_id
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_users__source_id
+#: model:ir.ui.menu,name:crm_campaign_configuration.menu_contact_source
+#: model:ir.ui.menu,name:crm_campaign_configuration.menu_crm_utm_source
+msgid "Source"
+msgstr ""

--- a/crm_campaign_configuration/i18n/es.po
+++ b/crm_campaign_configuration/i18n/es.po
@@ -1,0 +1,60 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* crm_campaign_configuration
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-05-28 08:49+0000\n"
+"PO-Revision-Date: 2021-05-28 08:49+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: crm_campaign_configuration
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_partner__campaign_id
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_users__campaign_id
+#: model:ir.ui.menu,name:crm_campaign_configuration.menu_contact_campaign
+#: model:ir.ui.menu,name:crm_campaign_configuration.menu_crm_utm_campaign
+msgid "Campaign"
+msgstr "Campaña"
+
+#. module: crm_campaign_configuration
+#: model:ir.model,name:crm_campaign_configuration.model_res_partner
+msgid "Contact"
+msgstr "Contacto"
+
+#. module: crm_campaign_configuration
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_partner__display_name
+msgid "Display Name"
+msgstr "Nombre mostrado"
+
+#. module: crm_campaign_configuration
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_partner__id
+msgid "ID"
+msgstr ""
+
+#. module: crm_campaign_configuration
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_partner____last_update
+msgid "Last Modified on"
+msgstr "Última modificación el"
+
+#. module: crm_campaign_configuration
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_partner__medium_id
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_users__medium_id
+#: model:ir.ui.menu,name:crm_campaign_configuration.menu_contact_medium
+#: model:ir.ui.menu,name:crm_campaign_configuration.menu_crm_utm_medium
+msgid "Medium"
+msgstr "Medio"
+
+#. module: crm_campaign_configuration
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_partner__source_id
+#: model:ir.model.fields,field_description:crm_campaign_configuration.field_res_users__source_id
+#: model:ir.ui.menu,name:crm_campaign_configuration.menu_contact_source
+#: model:ir.ui.menu,name:crm_campaign_configuration.menu_crm_utm_source
+msgid "Source"
+msgstr "Fuente"

--- a/crm_campaign_configuration/models/__init__.py
+++ b/crm_campaign_configuration/models/__init__.py
@@ -1,0 +1,1 @@
+from . import res_partner

--- a/crm_campaign_configuration/models/res_partner.py
+++ b/crm_campaign_configuration/models/res_partner.py
@@ -1,0 +1,14 @@
+# Copyright 2021 Berezi - Iker - AvanzOSC
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    campaign_id = fields.Many2one(
+        string='Campaign', comodel_name='utm.campaign')
+    medium_id = fields.Many2one(
+        string='Medium', comodel_name='utm.medium')
+    source_id = fields.Many2one(
+        string='Source', comodel_name='utm.source')

--- a/crm_campaign_configuration/views/contact_campaign_views.xml
+++ b/crm_campaign_configuration/views/contact_campaign_views.xml
@@ -1,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <menuitem id="menu_contact_campaign"
+            name="Campaign"
+            parent="contacts.res_partner_menu_config"
+            action="utm.utm_campaign_action"
+            sequence="50"/>
+</odoo>

--- a/crm_campaign_configuration/views/contact_medium_views.xml
+++ b/crm_campaign_configuration/views/contact_medium_views.xml
@@ -1,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <menuitem id="menu_contact_medium"
+            name="Medium"
+            parent="contacts.res_partner_menu_config"
+            action="utm.utm_medium_action"
+            sequence="50"/>
+</odoo>

--- a/crm_campaign_configuration/views/contact_source_views.xml
+++ b/crm_campaign_configuration/views/contact_source_views.xml
@@ -1,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <menuitem id="menu_contact_source"
+            name="Source"
+            parent="contacts.res_partner_menu_config"
+            action="utm.utm_campaign_action"
+            sequence="55"/>
+</odoo>

--- a/crm_campaign_configuration/views/res_partner_views.xml
+++ b/crm_campaign_configuration/views/res_partner_views.xml
@@ -1,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <field name="category_id" position="after">
+	            <field name="campaign_id"/>
+	            <field name="medium_id"/>
+	            <field name="source_id"/>
+            </field>
+        </field>
+    </record>
+</odoo>

--- a/crm_campaign_configuration/views/utm_campaign_views.xml
+++ b/crm_campaign_configuration/views/utm_campaign_views.xml
@@ -1,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <menuitem id="menu_crm_utm_campaign"
+            name="Campaign"
+            parent="crm.crm_menu_config"
+            action="utm.utm_campaign_action"
+            sequence="60"/>
+</odoo>

--- a/crm_campaign_configuration/views/utm_medium_views.xml
+++ b/crm_campaign_configuration/views/utm_medium_views.xml
@@ -1,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <menuitem id="menu_crm_utm_medium"
+            name="Medium"
+            parent="crm.crm_menu_config"
+            action="utm.utm_medium_action"
+            sequence="55"/>
+</odoo>

--- a/crm_campaign_configuration/views/utm_source_views.xml
+++ b/crm_campaign_configuration/views/utm_source_views.xml
@@ -1,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <menuitem id="menu_crm_utm_source"
+            name="Source"
+            parent="crm.crm_menu_config"
+            action="utm.utm_source_action"
+            sequence="50"/>
+</odoo>

